web_pwa/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Lorenzo Battistini @ TAKOBI
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Progressive web application",
+    "summary": "Make Odoo a PWA",
+    "version": "12.0.1.0.0",
+    "development_status": "Beta",
```

Comment:
Didn't you say alpha?

Reply:
@Yajo : why do you say that ? once reviewed, I think that the development status will be Beta. 
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_development_status.rst#1introduction
regards.

Reply:
I didn't say that :sweat_smile: 

See https://github.com/OCA/web/pull/1602#discussion_r427786417

Reply:
Didn't understood. "alpha" is for unstable module. Is it the case here ?

Reply:
I said "alpha" because for me the very 1st version of a module that nobody used in real life (AFAIK) and that likely will be refactored (see the discussion regarding sw) should be alpha.
That been said, @eLBati your call ;)

Reply:
I think Beta is more suitable for this